source/scripts/app.js

```diff
@@ -708,6 +727,9 @@ function bindCookbookCardButtons(card) {
     // Updates the COOKBOOK_TO_EDIT
     COOKBOOK_TO_EDIT = title;
     router.navigate("edit-cookbook");
+
+    // prefill edit form
+    fillEditCookbook(card.cookbookObj.title, card.cookbookObj.description);
```

Comment:
Use the getter functions instead of the properties directly
```suggestion
    fillEditCookbook(card.cookbook.title, card.cookbook.description);
```